Pinta.Core/Actions/ViewActions.cs

```diff
@@ -69,6 +70,7 @@ public ViewActions ()
             ToolBar = new ToggleCommand ("Toolbar", Translations.GetString ("Toolbar"), null, null);
             ImageTabs = new ToggleCommand ("ImageTabs", Translations.GetString ("Image Tabs"), null, null);
             PixelGrid = new ToggleCommand ("PixelGrid", Translations.GetString ("Pixel Grid"), null, Resources.Icons.ViewGrid);
+			StatusBar = new ToggleCommand ("Statusbar", Translations.GetString ("Statusbar"), null, null);
```

Comment:
I think maybe the label should be `Status Bar` rather than `Statusbar`?

Reply:
Good catch.  I looked at a few programs and there doesn't seem to be a consensus between `Statusbar` and `Status Bar`, but I think I prefer `Status Bar` too, so I committed a fix.